src/providers/hcloud/mod.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! aws ec2 metadata fetcher
```

Comment:
Leftover from ec2 provider.

Reply:
Thx, indeed missed that. Their metadata service seems largely EC2-compatible, so I started from that.